Add section aboute visibleCondition and disabledCondition (#548)
sulu/sulu-docs: book/templates.rst

        </properties>
    </template>

Visible/Disabled Conditions
---------------------------

> When you want to show/hide or disable a specific field by a condition

Reviewer: ```suggestion
It is possible to hide or disable specific properties based on the current values of the template.
```

> you can use the ``visibleCondition`` and ``disabledCondition`` attribute.

Reviewer: ```suggestion
the template by setting the `visibleCondition` and `disabledCondition` attribute.
```

> It uses `jexl`_ and you have access to all root properties of the resource.

Reviewer: ```suggestion
Inside of these attributes, you can utilize the `jexl`_ syntax for expressing your conditions.
```

---------------------------

It is possible to hide or disable specific properties based on the current values of
the template by setting the ``visibleCondition`` and ``disabledCondition`` attribute.

Inside of these attributes, you can utilize the `jexl`_ syntax for expressing your conditions.

.. code-block:: xml

    <!-- config/templates/pages/event.xml -->
    <?xml version="1.0" ?>
    <template xmlns="http://schemas.sulu.io/template/template"
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
        <!-- ... -->

        <properties>
            <!-- ... -->

            <property name="isCode" type="checkbox">
                <!-- ... -->
            </property>

            <property name="code" type="text_area" visibleCondition="isCode == true">
                <!-- ... -->
            </property>

            <property name="image" type="single_media_selection" disabledCondition="isCode != true">
                <!-- ... -->
            </property>

            <!-- ... -->
        </properties>
    </template>

.. note::

>     At current state you have inside the conditions only access to the root properties. So you can not

Reviewer: ```suggestion
    At the moment, conditions can only access the root properties of the template.
```

>     make conditions relative to a property inside a block.

Reviewer: ```suggestion
    This means that it is not possible to express conditions relative to a property inside a block.
```
